src/main/java/com/onaple/itemizer/data/serializers/MinerSerializer.java

```diff
@@ -38,7 +37,7 @@ public MinerBean deserialize(TypeToken<?> type, ConfigurationNode value) throws
                 inheritances.add(inheritance);
             }
         }
-        return new MinerBean(id, mineTypes, inheritances);
+        return null;//new MinerBean(id, mineTypes, inheritances);
```

Comment:
why always return null ? and if we always return null why we keep it ?

Reply:
This entire class will be removed,

Reply:
Remove all that should be, don't be shy!